python/ray/serve/tests/test_deploy_2.py

```diff
@@ -427,5 +427,25 @@ def check_health(self):
         time.sleep(0.1)
 
 
+def test_deploy_bad_working_dir_deployment(serve_instance):
```

Comment:
There's no `working_dir` in this test.

```suggestion
def test_deploy_bad_runtime_env_deployment(serve_instance):
```

Reply:
Whoops my bad, renamed!